### PR TITLE
Allows label constraints for issues on linked repos

### PR DESCRIPTION
**Pull request by discorick**

### CODE REVIEW

File: app/assets/stylesheets/_repo-settings.scss [hunk 1]
```diff
@@ -149,3 +149,26 @@
 
   }
 }
+
+div.hb-label-container {
```
Comment (rauhryan) [inline app/assets/stylesheets/_repo-settings.scss]: don't need the div.

File: app/assets/stylesheets/_repo-settings.scss [hunk 2]
```diff
@@ -149,3 +149,26 @@
...
+  text-align: center;
+
+  cursor: pointer;
+  -webkit-user-select: none;
+
+  &.selected {
+    box-shadow: inset 3px 3px 1px rgba(0,0,0,0.3);
+  }
+
+  span.label {
```
Comment (rauhryan) [inline app/assets/stylesheets/_repo-settings.scss]: don't need the span.

File: app/controllers/api/links_controller.rb [hunk 3]
```diff
@@ -47,7 +47,7 @@ def validate
         }, status: 403
       end
 
-      render json: { link: params[:link] }, status: 200
+      render json: { link: params[:link], labels: board.other_labels }, status: 200
```
Comment (rauhryan) [inline app/controllers/api/links_controller.rb]: name this other_labels

File: ember-app/app/components/settings/hb-label-chooser.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+var HbLabelChooserComponent = Ember.Component.extend({
+  classNames: ["hb-label-chooser"],
+  isVisible: Ember.computed.gt("labels.length", 0),
+  applyColor: function(){
+    if(this.get("labels.length")){
+      $("div.hb-label-container").each(function(i,el){
+        var color = Ember.$(el).data("color");
+        Ember.$(el).css("background", `#${color}`);
+        Ember.$(el).css("border-color", `#${color}`);
+      });
```
Comment (rauhryan) [inline ember-app/app/components/settings/hb-label-chooser.js]: doesn't feel like `ember` to me, we should prefer property binding to the style attribute

File: lib/bridge/github/labels.rb [hunk 5]
```diff
@@ -54,14 +54,16 @@ def destroy_label(name)
     alias_method :destroy_link, :destroy_label
 
 
-    def create_link(repo)
+    def create_link(repo, labels=nil)
       label_name = "Link <=> #{repo}"
-      match = Huboard.link_pattern.match label_name
+      label_name = labels ? "#{label_name} labels=#{labels.join('·')}" : label_name
```
Comment (rauhryan) [inline lib/bridge/github/labels.rb]: We should use some kind of UNICODE separator

File: ember-app/app/components/settings/hb-label-chooser.js (hunk 4 above)
Comment (discorick) [inline ember-app/app/components/settings/hb-label-chooser.js]: Ok, since this is feature flagged will revisit at a future date.

**Comment by discorick** [comment]
Preview of the final Product:

http://ilos.video/wnfuU1

**Comment by rauhryan** [comment]
:+1: to shipping

Noted the bug with special characters in labels in huboard/ghee#38

Fixing huboard/ghee#38 should fix the issue in HuBoard

Going to go ahead and merge

**Pull request merged**